SPEC.md

```diff
@@ -509,6 +509,54 @@ The behavior of this field depends on the version. Found in [Game End](#game-end
+| 0x23 | Ready To Start | uint8 | [0, 10] ranges as the "Ready to Start" banner swoops in during the VS menu. 0 means fully visible (ready to play). 10 means fully invisible (not ready to play) | CSS | 3.X.0
+| 0x24 | Stage Selected | uint8 | Stage ID for the stage that is currently hovered over in the SSS. | SSS | 3.X.0
+| 0x25 | P1 Controller Port Status | uint8 | 0 means `Plugged In`, 1 means `CPU`, 3 means `Unplugged`. | CSS | 3.X.0
+| 0x26 | P2 Controller Port Status | uint8 | 0 means `Plugged In`, 1 means `CPU`, 3 means `Unplugged`. | CSS | 3.X.0
+| 0x27 | P3 Controller Port Status | uint8 | 0 means `Plugged In`, 1 means `CPU`, 3 means `Unplugged`. | CSS | 3.X.0
+| 0x28 | P4 Controller Port Status | uint8 | 0 means `Plugged In`, 1 means `CPU`, 3 means `Unplugged`. | CSS | 3.X.0
+| 0x29 | P1 Character Selected | uint8 | The character ID currently selected (the one that shows up in the portrait) | CSS | 3.X.0
+| 0x2A | P2 Character Selected | uint8 | The character ID currently selected (the one that shows up in the portrait) | CSS | 3.X.0
+| 0x2B | P3 Character Selected | uint8 | The character ID currently selected (the one that shows up in the portrait) | CSS | 3.X.0
+| 0x2C | P4 Character Selected | uint8 | The character ID currently selected (the one that shows up in the portrait) | CSS | 3.X.0
+| 0x2D | P1 Coin Status | uint8 | 0 means `No Coin`, 1 means `Coin in Hand`, 2 means `Coin Down`, 3 means `Not Plugged In` | CSS | 3.X.0
+| 0x2E | P1 Coin Status | uint8 | 0 means `No Coin`, 1 means `Coin in Hand`, 2 means `Coin Down`, 3 means `Not Plugged In` | CSS | 3.X.0
+| 0x2F | P1 Coin Status | uint8 | 0 means `No Coin`, 1 means `Coin in Hand`, 2 means `Coin Down`, 3 means `Not Plugged In` | CSS | 3.X.0
+| 0x30 | P1 Coin Status | uint8 | 0 means `No Coin`, 1 means `Coin in Hand`, 2 means `Coin Down`, 3 means `Not Plugged In` | CSS | 3.X.0
+| 0x31 | Stage Select Cursor X | float | Stage select cursor x coordinate. [-27, 19] | SSS | 3.X.0
+| 0x35 | Stage Select Cursor Y | float | Stage select cursor y coordinate. [-27, 19] | SSS | 3.X.0
+| 0x39 | Frame Count | uint32 | Monotonically increasing frame number. Resets to 0 at most scene transitions. | All | 3.X.0
+| 0x3D | Sub-menu ID | uint8 | ID of sub-menu (`VS Mode`, `Name Entry`, `Language Select`, etc...) | All | 3.X.0
+| 0x3E | Menu Selection Index | uint8 | Index of the currently selected menu item. | Main Menus | 3.X.0
+| 0x3F | Online Costume Index | uint8 | Index of the currently selected online costume. | Online CSS | 3.X.0
+| 0x40 | Is Nametag Entry | uint8 | 5 means `True` | Online CSS | 3.X.0
+| 0x41 | P1 CPU Level | uint8 | [1, 9] CPU level set. | CSS | 3.X.0
+| 0x42 | P2 CPU Level | uint8 | [1, 9] CPU level set. | CSS | 3.X.0
+| 0x43 | P3 CPU Level | uint8 | [1, 9] CPU level set. | CSS | 3.X.0
+| 0x44 | P4 CPU Level | uint8 | [1, 9] CPU level set. | CSS | 3.X.0
+| 0x45 | P1 Is Holding CPU Slider | bool | 0 means `False`, 1 means `True` | CSS | 3.X.0
```

Comment:
should probably call this a `coin` like in line 540 or find a standard term we all agree with

Reply:
nvm, i'm an idiot